mscore/musescore.cpp

```diff
@@ -1702,7 +1702,7 @@ MuseScore::MuseScore()
       //---------------------
 
 #ifndef NDEBUG
```

Comment:
Seriously! Who came up with `#ifndef NDEBUG` rather than just `#ifdef DEBUG`?

Reply:
the C(++) standard... check the manual for assert()

Reply:
Wow! Talk about grandfathered-in! And I thought `if (NOT MSCORE_UNSTABLE)` was bad...